src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+$block_colors:
+  "1" hsl(0, 100%, 50%),
+  "2" hsl(0, 100%, 40%),
+  "3" hsl(0, 100%, 30%),
+  "4" hsl(0, 100%, 20%),
+  "5" hsl(0, 100%, 10%),
+  "6" #000;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```

Comment:
I would create a separate variable for block size, cause you use this value in several places.